### ISSUE TITLE
Chrome Extension Broken: caught Error: Extension context invalidated.  

### ISSUE THREAD

**Issue by KiranJKurian** [mukuljainx/Mokku#111]
Extension is broken and floods console. Disabling the extension stops the console from being flooded.

![Screenshot 2023-04-27 at 12 28 00 PM](https://user-images.githubusercontent.com/8814744/234929171-bcff0334-b287-497d-930d-cf4b0eb70367.png)

**Comment by quannt-paypay** [comment]
I'm seeing a bunch of `Could not establish connection. Receiving end does not exist` on my console. Even though the extension still works fine.

**Comment by Akaryatrh** [comment]
On my side it stills throw 2 errors even on private navigation (there's no other extension activated):
```
Uncaught Error: Couldn't find a style target. This probably means that the value for the 'insert' parameter is invalid.
    at c (panel.js:90:50464)
    at m (panel.js:90:51459)
    at l (panel.js:90:50157)
    at e.exports (panel.js:90:51791)
    at Object.<anonymous> (panel.js:244:8431)
    at n (panel.js:1:110)
    at Object.<anonymous> (panel.js:202:394061)
    at n (panel.js:1:110)
    at Object.<anonymous> (panel.js:202:392758)
    at n (panel.js:1:110)

localhost/:1 Uncaught (in promise) Error: Could not establish connection. Receiving end does not exist.
```

**Comment by mukuljainx** [comment]
Fixed with 2.0.0
